CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [Import/Disk] Fix `JSON.parse` and `createReadableSparseStream is not a function` errors [#6068](https://github.com/vatesfr/xen-orchestra/issues/6068)
+- [Backup]  Fix delta backup are almost always full backup instead of differencials [forum1](https://xcp-ng.org/forum/topic/5256/s3-backup-try-it/80) and [forum2](https://xcp-ng.org/forum/topic/5371/delta-backup-changes-in-5-66/24?_=1641289226655)
```

Comment:
```suggestion
- [Backup]  Fix delta backup are almost always full backup instead of differentials [Forum#5256](https://xcp-ng.org/forum/topic/5256/s3-backup-try-it/69) [Forum#5371](https://xcp-ng.org/forum/topic/5371/delta-backup-changes-in-5-66)
```